src/neuroconv/basedatainterface.py

```diff
@@ -8,7 +8,7 @@
 from .utils import get_base_schema, get_schema_from_method_signature
 
 
-class BaseDataInterface(ABC):
+class BaseDataInterface(object):
```

Comment:
Only thing about this is the `ABC` MRO can cause some metaclass conflicts with this approach

Reply:
Nevermind - figured out the resolution to the the metaclass conflict: https://stackoverflow.com/a/28727066